### PR TITLE
doc: clean-up documentation

### PR DESCRIPTION
**Pull request by Alex-Werner**
### Issue being fixed or implemented  

This is a fix for #108 instead of removing the links we cleaned up the examples folder in order to provide offline and online usage for client (Server part removed for now). 

### What was done  

- clean up documentation 
- removed duplicates

### CODE REVIEW

File: examples/client-usage-single-privateKey.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  privateKey: '9488797ee0018994d5ae22640e25210f65b1a450a0adcfa428cb5ffb29faa24b',
+});
+const account = wallet.getAccount();
+
+const start = async () => {
+  console.log('PrivateKey:', wallet.exportWallet());
+
+  const address = account.getUnusedAddress();
+  console.log('Unused Address :', address);
+
```
Comment (thephez) [inline examples/client-usage-single-privateKey.js:14]:
```suggestion
```

File: examples/offline-wallet-signing-message.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const wallet = new Wallet(walletOpts);
+const account = wallet.getAccount();
+
+const startSigningMessage = () => {
+  const message = new Message('Hello world!');
+
+  const idPrivateKey = account.getIdentityHDKey().privateKey;
+
+  const signed = account.sign(message, idPrivateKey);
+  const verify = message.verify(idPrivateKey.toAddress().toString(), signed.toString()); // true
```
Comment (thephez) [inline examples/offline-wallet-signing-message.js:19]: Is the comment on this line supposed to be removed?

Comment (Alex-Werner) [reply examples/offline-wallet-signing-message.js:19]: Not really, I mean, it's a way to know what the input looks like without having to run it. 
It's probably bad, may be it is sufficient to already have that info in the doc for message.verify in dashcore-lib ?

Comment (thephez) [reply examples/offline-wallet-signing-message.js:19]: I see. Probably fine either way.

**Pull request merged**